get-started/get-started-resources/get-started-resources.adoc

```diff
@@ -0,0 +1,47 @@
+= Get Started Resources
+:slug: get-started-resources
+:section: What is Neo4j
+:section-link: get-started
+
+= image:{img}icon-guide-theory.png[] Get Started Resources
+
+To help you in your learning path to learning more about Neo4j, we want to provide you with a few resources we used throughout this section, as well as a few additional resources for further information and development.
```

Comment:
edit: "To help you ~~in~~ **along** your ~~learning~~ path ~~to~~ **of** learning more about Neo4j, we want to provide you with a few **of the** resources we used throughout this section, as well as ~~a few~~ additional resources for further information and development."

Reply:
It might be because of the format I'm seeing this in, but it's unclear to me what section you're referring to? Everything else under "Get Started?" Did these resources appear in those sections?

Reply:
Yes, I was hoping to have a single place where people could see all the resources we linked to throughout the Get Started section. That way, they wouldn't have to sift through all the guides again to find a particular resource they needed. Does that make sense and do you think that's valuable?